packages/wdio-browserstack-service/src/Percy/PercyBinary.ts

```diff
@@ -0,0 +1,211 @@
+import url from 'node:url'
+import yauzl from 'yauzl'
+
+import { createRequire } from 'node:module'
+const require = createRequire(import.meta.url)
+
+const fs = require('node:fs')
+import fsp from 'node:fs/promises'
+const { https } = require('follow-redirects')
```

Comment:
Please don't use `require` anymore.
```suggestion
import fs from 'node:fs'
import fsp from 'node:fs/promises'
import { https } from 'follow-redirects'
```

Reply:
Done